logstash-core/src/test/java/org/logstash/execution/ShutdownWatcherExtTest.java

```diff
@@ -97,6 +116,13 @@ public void testShouldForceShutdown() throws InterruptedException {
                 Assertions.fail("Failed to shut down shutdown watcher thread");
             }
         }
+    }
+
+    private static String getPipelineTemplate() throws IOException {
+        return new String(Files.readAllBytes(PIPELINE_TEMPLATE));
+    }
 
+    private static String getPipeline(Boolean isDraining) throws IOException {
```

Comment:
Have a `getMethod` which is a read method to take a parameter, could be little bit counter intuitive.
I would suggest a rename like `resolvedPipeline` or something along those lines.